qrify/qrify

```diff
@@ -111,6 +111,35 @@ makeMultiLineQr()
   fi
 }
 
+# Function to get the json response from POST request
+function decodeQR {
```

Comment:
Why use the `function` syntax here?

Reply:
There is no real reason, I guess... I mostly use this syntax. If there is an advantage in using the other syntax, please let me know!

Reply:
The other parts of this package use it. And there is a mix of the two writings in this very script. (It‘s just a remark, nothing blocking at all.)

Reply:
There is no advantage one way or another but I would definitely prefer if you didn't include the ```function``` syntax only to keep the entire script that same :smile: